### ISSUE TITLE
A selector with a colon followed by a left brace should trigger an error or warning.

### ISSUE THREAD

**Issue by ghost** [CSSLint/csslint#559]
The following sample doesn't trigger any example or warning. Still the class is completely ignored by the browser (tested under the latest Chrome and Firefox).

```
.has-margin: {
  margin: 20px;
}
```

I don't know if it's "legal" in CSS to have this kind of construct, but if it is, I still think the end user should be warned about this kind of typo. Thanks.
